### DISCUSSION TITLE
Add support for FTP server

### DISCUSSION THREAD

**Discussion by louiemej** [adonisjs/core#2849] [Ideas]
Hi I would like to request to add ftp support for uploading and downloading files. 

I believe this is a good way to upload/download files when having multiple replica of Adonisjs instances (load balancing)

**Comment by bitkidd** [comment]
Adonis.js is a node.js framework, so you can use any npm package to do this.

**Comment by MadhuSaini22** [comment]
Currently the ACI ingress does not support FTP server (in passive mode). It would be very nice to have support for this in ACI to run small services like FTP directly in a container instance. Also support for a larger port range than the 10 ports currently supported.
